source/team/ways-of-working.html.md.erb

```diff
@@ -16,101 +16,49 @@ review_in: 6 months
-    * Closing an issue moves a task to __Done__, where moving an issue to __Done__ does not close it, hence close it, rather than move it.
-
-## Working with Git and GitHub
-
-There is a number of things we can do to make collaborating on code seamless.
-Although we do not want to impose too many rules on our team members, the following list is what we agreed as a team to do as a minimum:
-
-* Keeping a good git hygiene `git fetch`/`git pull` in `main` before branching out when starting on a new task. Merging `main` regularly into your branch on local and resolving the merge conflicts or rebasing before committing and pushing to remote.
-* Adding meaningful messages when committing, such as what have you achieved in this step.
-* Running tests, verifying if the tests/SCA has passed and that the `terraform plan` and the pipeline runs clean before creating a PR.
-* Providing enough information in the PR description so that another team member has enough context to be able to understand the change and to review it.
-Adding explanation on what is implemented with the PR and why. Adding information on tests if they were run outside of the GitHub actions pipeline and other useful links or references that help to understand the changes (e.g. a link to the issue or a doc).
-* Not approving PRs that do not have a good description or commit messages.
-* Approving only PRs that you understand (it applies to PRs created within and outside the team), limiting reviews to the platform code only.
-* Verifying the pipeline run clean post merge.
-
-We also think it is good to:
-
-* use a meaningful branch naming, using prefixes such as `feature/`, `fix/`, `docs/` etc., including an issue number and making it a summary of what your goal is (e.g. `feature/2563-usingOIDC-in-base`),
-* squash commits before committing to remote, so that each commit represents a step in the implementation (combining  planned changes with fixes to errors),
-* ... and/or squash commits when merging a remote branch with the `main`.
-
-### Tags and Releases
-
-* Using the [Semver](https://semver.org/) versioning strategy, prefixed with a `v` e.g. `v1.*.*` to `v2.0.0` would represent a backwards incompatible API change.
-* Providing a descriptive title for the release, so it's easily identifiable and understood.
-* Providing a high level summary of `What's New` in the release, so consumers can easily understand the changes that have been made - and potentially any changes they may need to make when incrementing to the new version number
-* Making use of GitHub automatically generated release notes. These provide in depth detail of the release.
-    * These can be heavily populated general maintenance PRs. It is worth refining the commit list down to commits that will impact consumers. There is a full changelog link provided for people who want the additional information.
-
-### Reviewing Contributor Changes to Maintained Modules
```

Comment:
Is there not value in keeping this section in the doc?

Reply:
I think I broadly cover that in these lines:
```
- Follow Semver for versioning
    - Provide a high-level `What's New` in the version release notes
    - Make use of GitHub's automatically generated release notes feature
```
But I'll add a little more detail in here!

Reply:
Personally I think lines 95 to 113 provide useful information that isn't covered by the replacement text. Why is that section being deleted?

Reply:
In the spirit of my review I chose to replace it with a more concise section, but as you've pointed out that section is useful to new starters - I've pushed a new commit that highlights it better.